buildSrc/src/main/java/com/uber/okbuck/core/model/android/AndroidAppTarget.java

```diff
@@ -52,12 +52,9 @@ public AndroidAppTarget(Project project, String name, boolean isTest) {
     Set<String> filters = ndkCompile != null ? ndkCompile.getAbiFilters() : ImmutableSet.of();
     cpuFilters = filters != null ? filters : ImmutableSet.of();
 
-    Boolean multidex = getBaseVariant().getMergedFlavor().getMultiDexEnabled();
-    if (multidex == null) {
-      multidexEnabled = false;
-    } else {
-      multidexEnabled = multidex;
-    }
+    multidexEnabled = Optional.ofNullable(getBaseVariant().getBuildType().getMultiDexEnabled())
+            .orElse(Optional.ofNullable(getBaseVariant().getMergedFlavor().getMultiDexEnabled())
+                    .orElse(false));
```

Comment:
seems like the mergedFlavour doesn't respect the multiDexEnabled value on the buildType. 

cc: @kageiit